### ISSUE TITLE
Invalid image for license agreement data

### ISSUE THREAD

**Issue by ruspl-afed** [eclipse-passage/passage#1042]
```
java.util.MissingResourceException: The image resource '../org.eclipse.passage.lic.agreements.edit/full/obj16/agreement.png' could not be located
	at org.eclipse.emf.common.util.DelegatingResourceLocator.delegatedGetImage(DelegatingResourceLocator.java:388)
	at org.eclipse.emf.common.util.DelegatingResourceLocator.getImage(DelegatingResourceLocator.java:314)
	at org.eclipse.passage.lic.licenses.edit.providers.AgreementDataItemProvider.getImage(AgreementDataItemProvider.java:197)
```

**Comment by ruspl-afed** [comment]
Closed with #1043

**Comment by ruspl-afed** [comment]
Still invalid

**Comment by ruspl-afed** [comment]
closed with #1044
